numpy/core/shape_base.py

```diff
@@ -304,6 +304,11 @@ def stack(arrays, axis=0):
     See Also
     --------
     concatenate : Join a sequence of arrays along an existing axis.
+    vstack : Stack arrays in sequence vertically, along the existing row axis.
+    hstack : Stack arrays in sequence horizontally, along the existing
+             column axis.
+    dstack : Stack arrays in sequence depth wise, along the existing
```

Comment:
I agree, should I change that in [`vstack`](https://github.com/numpy/numpy/blob/master/numpy/core/shape_base.py#L202), [`hstack`](https://github.com/numpy/numpy/blob/master/numpy/core/shape_base.py#L253) as well then?